cpu/ppc64_cpu_test.py

```diff
@@ -52,6 +53,9 @@ def setUp(self):
         self.smt_values = {1: "off"}
         self.key = 0
         self.value = ""
+        self.max_smt_value = 4
+        if cpu.get_cpu_arch().lower() == 'power8':
+            self.max_smt_value = 8
```

Comment:
i think should be fine as  power7 and lower architecture  will not support smt level 8 it is just 4 , and other arch this test not supported